packages/core/src/new-api/internal/journal/utils/serialize-replacer.ts

```diff
@@ -0,0 +1,34 @@
+import { serializeError } from "serialize-error";
+
+/**
+ * When stringifying messages to the journal, this defines the replacer.
+ */
+export function serializeReplacer(_key: string, value: unknown) {
+  if (value instanceof Set) {
+    return Array.from(value).sort();
+  }
+
+  if (value instanceof Map) {
+    return Object.fromEntries(value);
+  }
+
+  if (typeof value === "bigint") {
+    return `${value.toString(10)}n`;
```

Comment:
This feels a bit risky, not too risky, but we could do something with less changes of clashing, like `{$bigint: "123"}`.

Reply:
I switched to use the same serialization as in the `stored-deployment-serializer` so `{_kind: "bigint", value: n.toString(10)}`.